Apps/dozzle-swarm/docker-compose.yml

```
@@ -0,0 +1,70 @@
+        target: 8080
+        published: "8888"
+        protocol: tcp
+    restart: unless-stopped
+    volumes:
+      - type: bind
+        source: /var/run/docker.sock
+        target: /var/run/docker.sock
+        bind:
+          create_host_path: true
+    environment:
+      - DOZZLE_MODE=swarm      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)